xds/src/main/java/io/grpc/xds/WeightedRoundRobinLoadBalancer.java

```diff
@@ -258,7 +259,8 @@ final class WeightedRoundRobinPicker extends RoundRobinPicker {
         new HashMap<>();
     private final boolean enableOobLoadReport;
     private final float errorUtilizationPenalty;
-    private volatile EdfScheduler scheduler;
+    private volatile EdfScheduler edfScheduler;
+    private volatile StaticStrideScheduler ssScheduler; // what does volatile mean?
```

Comment:
volatile means the object is thread safe: if the object is accessed from different thread, JVM will make sure one operation at a time. This is needed due to `pick()` operation can be invoked from different threads.